Run3 tt+X (TOP) cards (Madgraph5_aMC@NLO)

Cvico opened the pull request:
Dear experts, 

I'm a MC contact for the TOP group. We are starting to work on the Run3 TOP MC
requests, and thus we would like to have the cards included in this PR reviewed and, 
if possible, merged into the master branch.

This PR includes:
-  Slightly modified version of the cards used in Run2 tt+X production. Modifications include:
        - Center-of-mass energy adjusted to 6.8 TeV per beam.
        - Typo **nevents_for_max_weigth** has been fixed in all madspin cards.

Please let me know if you have any comment or question.

Best regards,
Carlos Vico

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_Dilept_5f_ckm_LO/ttGamma_Dilept_5f_ckm_LO_proc_card.dat:
```diff
@@ -0,0 +1,15 @@
+define j = j b b~
+define part =  ve vm vt e- mu- ve~ vm~ vt~ e+ mu+ ta- ta+
+generate p p > t t~ a, t > b part part, t~ > b~ part part  @1
+add process p p > t t~, t > b part part a, t~ > b~ part part  @2
+add process p p > t t~, t > b part part , t~ > b~ part part a  @3
```
have you checked the overlap between line 9 and line10/11

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_Dilept_5f_ckm_LO/ttGamma_Dilept_5f_ckm_LO_run_card.dat:
```diff
@@ -0,0 +1,269 @@
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  250000 = nevents ! Number of unweighted events requested 
```
change "250000 = nevents" to a small number, 10000

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromT_5f_ckm_LO/ttGamma_SingleLeptFromT_5f_ckm_LO_proc_card.dat:
```diff
@@ -0,0 +1,16 @@
+define partl =  ve vm vt e- mu- ve~ vm~ vt~ e+ mu+ ta- ta+
+define partq =  u c d s u~ c~ d~ s~
+generate p p > t t~ a, t > b partl partl, t~ > b~ partq partq  @1
+add process p p > t t~, t > b partl partl a, t~ > b~ partq partq  @2
+add process p p > t t~, t > b partl partl , t~ > b~ partq partq a  @3
```
same here

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromT_5f_ckm_LO/ttGamma_SingleLeptFromT_5f_ckm_LO_run_card.dat:
```diff
@@ -0,0 +1,269 @@
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  250000 = nevents ! Number of unweighted events requested 
```
change "250000 = nevents" to 10000

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromTbar_5f_ckm_LO/ttGamma_SingleLeptFromTbar_5f_ckm_LO_proc_card.dat:
```diff
@@ -0,0 +1,16 @@
+define partl =  ve vm vt e- mu- ve~ vm~ vt~ e+ mu+ ta- ta+
+define partq =  u c d s u~ c~ d~ s~
+generate p p > t t~ a, t > b partq partq, t~ > b~ partl partl  @1
+add process p p > t t~, t > b partq partq a, t~ > b~ partl partl  @2
+add process p p > t t~, t > b partq partq , t~ > b~ partl partl a  @3
```
same here

menglu21 commented on bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/ttGamma_SingleLeptFromTbar_5f_ckm_LO/ttGamma_SingleLeptFromTbar_5f_ckm_LO_run_card.dat:
```diff
@@ -0,0 +1,269 @@
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  250000 = nevents ! Number of unweighted events requested 
```
change "250000 = nevents" to "10000 = nevents"

Cvico commented:
Hi,

Thank you very much for the comments. I think with this PR I have addressed all of them.

Regarding the ttgamma samples, indeed those cards were outdated. I accidentally copied an older
version of the cards that indeed was shown to introduce double counted diagrams because of overlapping
lines 9,10 and 11 in the process card. I've included the correct ones now, and removed the old ones from the
13p6TeV folder so that they are not used by mistake.

Best regards,
Carlos

sihyunjeon commented:
TTW, TTZ, TTG is in the list of central run3 sample production plans.

can you check this with gen convenors

menglu21 commented:
@sihyunjeon thanks for the reminder, i overlooked it. @Cvico those samples will be handled centrally, so we will put this PR on hold and make them consistent with the cards used in central production before merging.
